bin/ci-flake-detect

```diff
@@ -6,9 +8,7 @@
 # As of the Change Date specified in that file, in accordance with
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
+#
+# ci-flake-detect - Detect known flakes based on Github tags
```

Comment:
I do not think "Flake" is the correct term to use here and further down. I think it should be something along the lines of `LoggedError` or another similar term, because in at least 50% of the cases those are actual product bugs and not test issues.

We as a company unfortunately already made a linguistic crime by the widespread use of the word "flake" (even the GitHub tag is named `ci-flake` :-( ). People say "test X is flaky", which pins the blame squarely on the test and not on the code, even in cases where there is an obvious product bug with a panic in the logs. In our messaging as a QA team we should try to counter that tendency by all available means.